### PR TITLE
Adds a configMap named `hub-info` to have the Hub version

### PR DESCRIPTION
**Pull request by PuneetPunamiya**
Signed-off-by: Puneet Punamiya ppunamiy@redhat.com

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/main/standards.md#principles) (if functionality changed/added)
- [ ] Run API Unit Tests, Lint Checks, API Design, Golden Files with `make api-check`
- [ ] Run UI Unit Tests, Lint Checks with `make ui-check`
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/main/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/main/CONTRIBUTING.md) for more details._

# Release Notes

### CODE REVIEW

File: config/07-hub-info/hub-info.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2023 The Tekton Authors
```
Comment (vinamra28) [inline config/07-hub-info/hub-info.yaml:1]:
```suggestion
# Copyright 2024 The Tekton Authors
```

File: config/07-hub-info/hub-info.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hub-info
+  labels:
+    app.kubernetes.io/instance: default
```
Comment (vinamra28) [inline config/07-hub-info/hub-info.yaml:20]: why default?

File: config/07-hub-info/hub-info.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hub-info
```
Comment (vinamra28) [inline config/07-hub-info/hub-info.yaml:18]: shouldn't we fetch this from the namespace where hub components are installed?

File: config/07-hub-info/role-binding.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+# Copyright 2023 The Tekton Authors
```
Comment (vinamra28) [inline config/07-hub-info/role-binding.yaml:1]:
```suggestion
# Copyright 2024 The Tekton Authors
```

File: config/07-hub-info/hub-info.yaml (hunk 3 above)
Comment (PuneetPunamiya) [reply config/07-hub-info/hub-info.yaml:18]: you mean `tekton-hub-info` ?

**Review by vinamra28 — APPROVED** [review]
/lgtm

**Pull request merged**